pontoon/base/admin.py

```diff
@@ -55,6 +59,44 @@ def save_model(self, request, obj, form, change):
 
             log_user_groups(request.user, obj, (add_groups, remove_groups))
 
+    def delete_model(self, request, obj):
+        random_hash = uuid.uuid4().hex
+        new_user = User.objects.create_user(
+            username="deleted-user-" + random_hash,
+            email="deleted-user-" + random_hash + "@example.com",
+            first_name="Deleted User",
```

Comment:
Let's also set [`is_active`](https://docs.djangoproject.com/en/3.0/ref/contrib/auth/#django.contrib.auth.models.User.is_active) flag to `False`. That way it will be easier to filter deleted users.

Reply:
Added it